Microsoft.Toolkit.Uwp.SampleApp/SamplePages/FlexPanel/FlexPanelExperimentCode.bind

```diff
@@ -0,0 +1,129 @@
+    xmlns:local="using:FlexPanelTest"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Page.Content>
+        <Grid Margin="10, 0">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="*" />
+            </Grid.RowDefinitions>
+            <Grid.Children>
+                <!-- Control Panel -->
+                <Grid 
+              Grid.Row="0">
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="Auto" />
+                        <RowDefinition Height="Auto" />
+                        <RowDefinition Height="Auto" />
+                        <RowDefinition Height="Auto" />
+                        <RowDefinition Height="Auto" />
+                        <RowDefinition Height="Auto" />
+                    </Grid.RowDefinitions>
+
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto" />
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+
+                    <Grid.Resources>
+                        <ResourceDictionary>
+                            <Style TargetType="TextBox">
+                                <Style.Setters>
+                                    <Setter Property="VerticalAlignment" Value="Center" />
+                                </Style.Setters>
+                            </Style>
+                        </ResourceDictionary>
+                    </Grid.Resources>
+                    <Grid.Children>
+                        <TextBlock 
+                Text="Number of items:"
+                VerticalAlignment="Center"       
+                Grid.Row="0" Grid.Column="0" />
+
+                        <StackPanel 
+                Orientation="Horizontal"
+                Spacing="10"
+                Grid.Row="0" Grid.Column="1">
+
+                            <!--TextBlock 
+                    Text="{x:Bind Path=numberStepper.Value }" 
+                    VerticalAlignment="Center"       
+                    /-->
+
+                            <local:Stepper x:Name="numberStepper"
+                         MinValue="0"
+                         MaxValue="99"
+                         Increment="1"
+                         Value="3"
+                         ValueChanged="OnNumberStepperValueChanged" />
+
+                        </StackPanel>
+
+                        <TextBlock 
+                Text="Direction:"
+                Grid.Row="1" Grid.Column="0" 
+                VerticalAlignment="Center"       
+                />
+                        <local:EnumPicker EnumType="controls:FlexDirection"
```

Comment:
All of these options should be part of the sample binding. Add a .bind file and change the FlexPanel to bind these properties 
Example: 
```
<FlexPanel Direction="@[Direction:Enum:FlexDirection.Row]"
           Wrap="@[Wrap:Enum:FlexWrap.NoWrap]"
           AlignItems="@[Item Alignment:Enum:FlexAlignItems.Start]"
```

Reply:
Again, take a look at the update that was pushed at about the same time you made these comments.